modules/ibi-installer-configuration-config.adoc

```diff
@@ -45,4 +45,12 @@ The name of the interface must match the actual NIC name as shown in the operati
 
 |`releaseRegistry`|`string`| Specifies the container image registry that you used for the release image of the seed cluster. 
 
+|`nodeLabels` |`string`| Specifies the node labels for the {sno} node, for example:
```

Comment:
I wonder should not we mention that those are like custom or additional node labels? If not this is great

Reply:
Sounds good... updated to "Specifies custom node labels..." 
Hopefully this is clear that they are additional and custom